content/02.intro.md

```diff
@@ -1,13 +1,27 @@
-These profiling methods, including RNA sequencing (RNA-seq, whole genome sequencing, imaging data, electronic health record data, among others, generate large and complex data. 
-The analysis of such complex data from rare disease will require machine learning (ML)-based methodologies to assist in the modeling and interpretation of this data. 
-Indeed, a systematic review of application of ML in rare disease in the last 10 years uncovered 211 human data studies in 74 different rare diseases employing ensemble methods (36.0%), support vector machines (32.2%) and artificial neural networks (31.8%) [@doi:10.1186/s13023-020-01424-6]. 
-While this review points to the increasing popularity of ML methods in rare disease, there are various hurdles that are inherent to such datasets.
-ML based methods benefit from large sample sizes, rare disease datasets typically contain fewer than one hundred samples [@doi:10.1186/s13023-020-01424-6].
+Rare disease research is increasingly dependent on high-throughput profiling of samples and would greatly benefit from machine learning (ML) analytics. 
+Machine learning algorithms are computational methods that can identify patterns in data, and can use information about these patterns to perform tasks (e.g., pick out important data points or predict outcomes when they are not yet known, among other tasks). 
+A systematic review of application of ML in rare disease uncovered 211 human data studies that used ML to study 74 different rare diseases over the last 10 years [@doi:10.1186/s13023-020-01424-6]. 
+Indeed, ML can be a powerful tool in biomedical research but it does not come without pitfalls [TODO: ref], some of which are magnified in a rare disease context.
+In this perspective, we will focus our discussion on considerations for two types of ML in the context of the study of rare diseases: supervised and unsupervised learning.
+
+_Supervised algorithms_ require training data with specific phenotype labels (e.g., responder vs. non-responder) and learn correlations of features with the phenotype labels to predict the phenotype labels of unseen or new test data (e.g., predicting which patients did or did not respond to treatment).
+If the goal of a study is to classify patients with a rare disease into molecular subtypes based on high-throughput profiling, researchers would select a supervised ML algorithm to carry out this task. 
+A supervised ML model is of limited utility if it can only accurately predict phenotype labels in the data it was trained on (this is called _overfitting_); instead, it's more beneficial to develop models that _generalize_ or maintain performance when applied to new data that has not yet been "seen" by the model.
+In later sections, we'll discuss regularized models, a strategy for reducing overfitting that can be useful for rare disease datasets. 
+
+_Unsupervised algorithms_ can learn patterns or features from _unlabeled_ training data.
```

Comment:
It might be better to tackle the unsupervised first and then move to supervised.

Reply:
Works for me!

Reply:
ok.. will merge this one and start a fresh PR :)